browser/ui/views/tabs/brave_new_tab_button.h

```diff
@@ -19,6 +19,10 @@ class BraveNewTabButton : public NewTabButton {
   private:
     gfx::Size CalculatePreferredSize() const override;
     void PaintPlusIcon(gfx::Canvas* canvas, int offset, int size) override;
+    SkPath GetNewerMaterialUiButtonPath(float button_y,
+                                        float scale,
+                                        bool extend_to_top,
+                                        bool for_fill) const override;
```

Comment:
Should this be overridden? It's not marked virtual in the parent and won't build.

Reply:
@jasonrsadler it is marked virtual, via a patch

Reply:
:1 I didn't have that. Thanks.